Gopkg.toml

```diff
@@ -29,3 +29,11 @@
 [[constraint]]
   name = "github.com/stretchr/testify"
   version = "1.2.2"
+
+[[constraint]]
+  branch = "master"
```

Comment:
Pin to a commit/branch/tag instead just `master`?

Reply:
Also, in Cagent it needs to be pinned, because Cagent vendored a version of master that has been rewinded.